webrtc.html

```diff
@@ -5842,8 +5831,8 @@ <h2>Methods</h2>
                       <p>Queue a task that runs the following steps:</p>
                       <ol>
                         <li>
-                          <p>If <var>connection</var>'s <a>[[\IsClosed]]</a>
-                          slot is <code>true</code>, abort these steps.</p>
+                          <p>If <var>transceiver</var>'s <a>[[\Stopped]]</a> slot is
+                          <code>true</code>, abort these steps.</p>
```

Comment:
Why not reject p?
(I know it said "abort" before. Leaving dangling promises when we don't have to seems ugly.)

Reply:
According to Jan-Ivar, we're being consistent that closed objects drop promises rather than rejecting them.